project1-sps/script.js

```diff
@@ -0,0 +1,155 @@
+//Objective SPS human vs computer
+// What is the input going to be? scissors, paper, stone
+// What should the output be?
+//scissors beats paper,
+//paper beats stone, and
+//stone beats scissors
+//same will be draw
+// How will the computer's option be randomly generated? math.random
+// How many different cases are there?
+// Scissors Paper Stone
+// 0        0     0 (not possible)
+// 0        0     1 (YES, BOTH STONE)
+// 0        1     0 (YES, BOTH PAPER)
+// 0        1     1 (YES)
+// 1        0     0 (YES, BOTH SCISSORS)
+// 1        0     1 (YES)
+// 1        1     0 (YES)
+// 1        1     1 (not possible)
+
+// function to trigger button click on enter key
+document
+  .getElementById("input-field")
+  .addEventListener("keyup", function (event) {
+    event.preventDefault();
+    if (event.keyCode === 13) {
+      document.getElementById("submit-button").click();
+    }
+  });
+
+// function to generate random number
+var generateRandomNumber = function () {
+  // produces a decimal between 0 and 6
+  var randomDecimal = Math.random() * 6;
+
+  // take off the decimal
+  var randomInteger = Math.floor(randomDecimal);
+
+  // it's a number from 0 - 5 ... add 1
+  var randomNumber = randomInteger + 1;
+
+  console.log(`===== random generated number is ${randomNumber} =====`);
+
+  return randomNumber;
+};
+
+//set the initial Scissor Paper Stone (SPS) state  as blank
+//(??? Q: is this necssary and must it be at global scope?)
+var spsState = "";
```

Comment:
i think from the way you designed it, its not necessary for it to be a global variable and it can just be local within `convertRandomNumberToSpsState`